sdk/search/azure-search/out/azure/search/index/_index_batch.pyi

```diff
@@ -0,0 +1,13 @@
+from ._generated.models import IndexAction as IndexAction
+from typing import Any, List
+
+def flatten_args(args: Any): ...
+
+class IndexBatch:
+    def __init__(self) -> None: ...
+    def add_upload_documents(self, *documents: Any) -> None: ...
+    def add_delete_documents(self, *documents: Any) -> None: ...
+    def add_merge_documents(self, *documents: Any) -> None: ...
+    def add_merge_or_upload_documents(self, *documents: Any) -> None: ...
+    @property
+    def actions(self) -> List[IndexAction]: ...
```

Comment:
Observation; this allows me to write code like:

```python
batch = IndexBatch()
batch.actions.append(...) # Doesn't raise any exceptions, but the appended stuff is lost since batch.actions returned a copy...
```
Would it make sense to return an iterator instead?